### DISCUSSION TITLE
Rollback tenant migration

### DISCUSSION THREAD

**Discussion by ghost** [spatie/laravel-multitenancy#342] [Q&A]
Hi,
Whenever I try to rollback a migration I get the following error:
```
  SQLSTATE[HY000] [2002] Operation timed out (SQL: select max(`batch`) as aggregate from `migrations`)
```
My tenant connection is working, I can fetch/ store data from the tenant database without any problem, but when I try to rollback a migration there's a connection timeout.
I use the following command:
```
tenants:artisan "migrate:rollback --database=tenant"
```
I use a similar command for migrating the tenants (`migrate --step --database=tenant`) and then it's working fine. What am I doing wrong here?

**Comment by masterix21** [comment]
@thatsalltheyare is not easy to reply because the error is an `Operation timed out`: seems not related to the package.
